### DISCUSSION TITLE
Support for Rumble.com links?

### DISCUSSION THREAD

**Discussion by SpinchGang** [WaterMediaTeam/watermedia#104] [Ideas]
Sometimes youtube links break and don't work, so having rumble links work as a backup option would be great.

**Comment by SrRapero720** [comment]
rumble seems... amm... odd... not a quite large entertainment platform

i will let this open, in case more péople request it to make a patch for v2.1, otherwise i'll delegate to v3

**Comment by SrRapero720** [comment]
Not posible and not planned
